src/test/java/edu/ucla/library/services/metadata/MetadataSetterTest.java

```diff
@@ -32,7 +39,12 @@
     /**
      * Path to media files to be read.
      */
-    private static final String MEDIA_PATH = "src/test/resources/media/";
+    private static final String MEDIA_PATH = TEST_FIXTURES_DIR + "media/";
+
+    /**
+     * A secondary path from which media files can be read.
```

Comment:
I think this comment should explain that it's a comma-delimited string of paths (maybe also have another constant for the second path)

Reply:
You mean the comment for line 47?

Reply:
Yeah, the comment for line 47. I think the GitHub UI is just showing line 42-44 for context (I had only selected line 45 when I made this comment).

Reply:
Comment updated to make it clearer that the path should be a comma delimited string.